core/src/main/java/org/secondbase/core/SecondBase.java

```diff
@@ -1,45 +1,26 @@
 package org.secondbase.core;
 
-import java.util.ServiceLoader;
 import org.secondbase.core.config.SecondBaseModule;
-import org.secondbase.core.moduleconnection.WebConsole;
 import org.secondbase.flags.Flags;
 
 /**
  * Coordination class for SecondBase modules. Handles Flags parsing and cooperation between modules.
  */
 public class SecondBase {
 
-    private Flags flags;
-    private WebConsole webConsole;
-
-    private final ServiceLoader<SecondBaseModule> configurableModule
-            = ServiceLoader.load(SecondBaseModule.class);
 
-    /**
-     * Set up SecondBase with default values.
-     * @param args command line arguments
-     */
-    public SecondBase(final String[] args) throws SecondBaseException {
-        init(args, new Flags());
-    }
+    private Flags flags;
 
-    /**
-     * Set up SecondBase.
-     * @param args command line arguments
-     * @param flags preloaded Flags class
-     */
-    public SecondBase(final String[] args, final Flags flags) throws SecondBaseException {
-        init(args, flags);
+    public SecondBase(final SecondBaseModule[] modules) throws SecondBaseException{
```

Comment:
Sorry, I mistyped the constructor example. There is never any case where the user will not use args in any way. Either he has his own Flags or he has modules which has flags. So I think we should have these two constructors:

SecondBase(final String[] args, final SecondBaseModule[] modules)
SecondBase(final String[] args, final SecondBaseModule[] modules, final Flags flags)